source/test/webextensionPolyfill.js

```diff
@@ -0,0 +1,3 @@
+import browser from "webextension-polyfill";
+
+globalThis.browser = browser;
```

Comment:
Required because the polyfill does not automatically set the `browser` global when bundled.

https://github.com/mozilla/webextension-polyfill/issues/350